src/contrib/talloc.rst

```diff
@@ -253,11 +253,25 @@ running process:
         -ex 'detach' \
         -ex 'quit' &> /dev/null
 
+SSSD-2.6.0 and above do not use the deprecate ``talloc_autofree_context()``
+main context anymore but a dedicated local context called ``autofree_ctx``. As
+a result the following snippet should be used:
+
+.. code-block:: bash
+
+    PROCESS=$(pidof sssd_nss)
+    FILE=/tmp/talloc.$PROCESS
+    sudo gdb -quiet -batch -p $PROCESS \
+        -ex "set \$file = (FILE*)fopen(\"$FILE\", \"w+\")" \
+        -ex 'call talloc_report_full(autofree_ctx, $file)' \
```

Comment:
Hi,

thanks for the hint, I added `(void)` so that libtalloc debug information is not required. I added a comment as well that glibc and SSSD debug information are needed for `FILE` and `autofree_ctx` respectively.

bye,
Sumit